starboard/common/thread.h

```diff
@@ -34,11 +34,18 @@ class Semaphore;
 class Thread {
  public:
   explicit Thread(const std::string& name);
+  Thread(const std::string& name, int64_t stack_size);
   template <size_t N>
   explicit Thread(char const (&name)[N]) : Thread(std::string(name)) {
```

Comment:
Use the constructor you added below and you'll be able to remove the static assert in the body here so it's not duplicated